hw/top_earlgrey/sw/autogen/top_earlgrey.h

```diff
@@ -772,6 +772,14 @@ typedef enum top_earlgrey_power_manager_wake_ups {
   kTopEarlgreyPowerManagerWakeUpsLast = 0, /**< \internal Last valid pwrmgr wakeup signal */
 } top_earlgrey_power_manager_wake_ups_t;
 
+/**
+ * Power Manager Reset Request Signals
+ */
+typedef enum top_earlgrey_power_manager_reset_requests {
+  kTopEarlgreyPowerManagerResetRequestsNmiGenNmiRstReq = 0, /**<  */
```

Comment:
If this is going to apply to both power manager and reset manager, it is probably a good idea to remove "power manager" from the name, as @lenary suggests.